lib/pyld/documentloader/requests.py

```diff
@@ -60,7 +60,8 @@ def loader(url, options={}):
                 headers = {
                     'Accept': 'application/ld+json, application/json'
                 }
-            response = requests.get(url, headers=headers, **kwargs)
+                kwargs['headers'] = headers
```

Comment:
I don't think this patch will fix the issue. From what I understand there are 2 places where headers are retrieved. One is the "options" parameter in `def loader(url, options={})` and the other one is from the "kwargs" in `def requests_document_loader(secure=False, **kwargs)`. kwargs is used to pass user arguments to `requests`, and adding `                kwargs['headers'] = headers` will simply remove/replace any headers. It should probably be a merge of the "headers" speficied in `def loader(url, options={})` and `def requests_document_loader(secure=False, **kwargs)`.